bin/submitIndependentControlWorkflows.sh

```diff
@@ -56,7 +56,7 @@ else
     touch $submissionMarker
 fi
 
-currentJobs=$(bjobs -w | grep $controlJobSuffix)
+currentJobs=$(squeue -o "%.18i %.9P %.8j %.8u %.2t %.10M %.6D %R" | grep $controlJobSuffix)
```

Comment:
Here, you can write all `-o %all` to output all possible info, or simply remove the column size settings to ensure that you capture the entirety of any info, e.g.,
```suggestion
currentJobs=$(squeue -o "%i %P %j %u %t %M %D %R" | grep $controlJobSuffix)
```